cmd/nodecmd/local.go

```diff
@@ -477,16 +474,15 @@ func localValidate(_ *cobra.Command, args []string) error {
 		return err
 	}
 
-	extraAggregatorPeers, err := blockchain.GetAggregatorExtraPeers(app, clusterName, []string{})
+	extraAggregatorPeers, err := blockchain.GetAggregatorExtraPeers(app, clusterName)
 	if err != nil {
 		return err
 	}
 	aggregatorLogger, err := utils.NewLogger(
 		constants.SignatureAggregatorLogName,
-		aggregatorLogLevel,
+		flags.SigAggFlags,
 		constants.DefaultAggregatorLogLevel,
```

Comment:
why do we pass in the default log level here if we're already including the flag?

Reply:
yup, addressed